index.js

```diff
@@ -31,6 +31,10 @@ module.exports = input => {
 		throw new Error('You should not publish when running Node.js 6. Please downgrade and publish again. https://github.com/npm/npm/issues/5082');
 	}
 
+	if (!opts.anyBranch && execa.sync('git', ['symbolic-ref', '--short', 'HEAD']).stdout !== 'master') {
+		throw new Error('Not on "master" branch. Use --any-branch to publish anyway.');
```

Comment:
Sure. I was on the fence about it. My reasoning was that it is a literal value and backticks don't do anything special in this context. I did use them in the readme. Will put them here, too.